inc/authldap.class.php

```diff
@@ -3945,4 +4003,31 @@ public static function getUsers($values, &$results, &$limitexceeded) {
 
       return $users;
    }
+
+   public function checkFilesExist(&$input) {
+
+      if (isset($input['tls_certfile'])) {
+         $file = realpath($input['tls_certfile']);
+         if (!file_exists($file)) {
+            Session::addMessageAfterRedirect(
+               __('TLS certificate path is incorrect'),
+               false,
+               ERROR
+            );
+            return false;
+         }
+      }
+
+      if (isset($input['tls_keyfile'])) {
```

Comment:
I guess we should allow removal of field value.
```suggestion
      if (isset($input['tls_keyfile']) && !empty($input['tls_keyfile'])) {
```